ui/app/appointments/views/manage/list/listView.html

```diff
@@ -19,20 +19,19 @@
                         <i class="fa fa-caret-down" ng-show="!reverseSort"></i>
                 </span>
                 </th>
-                <th ng-show="searchedPatient">{{::'APPOINTMENT_DATE' | translate}}</td>
```

Comment:
@amosh101 what is the reason fo removing this tag?

Reply:
The date was returned in the last column previously, but since it was requested to come before the block column the change was done in the variable table info, that handles the other table head fields.

Reply:
ok great